Assets/Scripts/Lever.cs

```diff
@@ -2,24 +2,35 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Events;
+using UnityEngine.Assertions;
+
+[RequireComponent(typeof(Animator))]
 
 public class Lever : MonoBehaviour {
-    public UnityEvent leftOption;
-    public UnityEvent rightOption;
+    public UnityEvent leftMode;
+    public UnityEvent rightMode;
     private bool left;
     private Animator animator;
     private bool inRange;
 
-    void Awake()
+    private void Start()
     {
         left = true;
         inRange = false;
-        animator = gameObject.GetComponent<Animator>();
+        animator = GetComponent<Animator>();
+        Assert.IsNotNull(animator);
```

Comment:
Can you add the message in case it the assertion actually triggers? Also can you comment why it should be not null (i.e., because of the `RequireComponent` part)?